lib/ramble/ramble/software_environments.py

```diff
@@ -416,10 +412,6 @@ def all_packages(self):
 class SoftwareEnvironments(object):
     """Class representing a group of software environments"""
 
-    _deprecated_sections = [namespace.variables, namespace.zips,
```

Comment:
In addition to removing them from here, these should be removed from the schema too: https://github.com/GoogleCloudPlatform/ramble/blob/develop/lib/ramble/ramble/schema/spack.py#L44-L48

Reply:
Ah right, sorry for the omission. Removed.